cooltools/expected.py

```diff
@@ -221,6 +201,97 @@ def count_all_pixels_per_diag(n):
     return np.arange(n, 0, -1)
 
 
+def count_all_pixels_per_block(clr, supports):
```

Comment:
`count_all_pixels_per_block`
`count_all_pixels_per_diag`

`count_bad_pixels_per_block`
`count_bad_pixels_per_diag`

such a naming might be misleading, because it hints a unified API, which is not - `per_diag` ones are unaware of `cooler/supports` - they deal with matrix dimensions, while `per_block` are `clr/supports` based

but consistent naming is kind of nice otherwise ...